applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_analysis.py

```diff
@@ -92,7 +92,7 @@ def _SetUpGiDOutput(self):
         if self.parallel_type == "OpenMP":
             from gid_output_process import GiDOutputProcess as OutputProcess
         elif self.parallel_type == "MPI":
-            from gid_output_process_mpi import GiDOutputProcessMPI as OutputProcess
+            from KratosMultiphysics.mpi.distributed_gid_output_process import DistributedGiDOutputProcess as OutputProcess
```

Comment:
I don't think this will ever be used
Somehow beats the purpose of PotentialFlow :D